### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by luyaozou** [Ltotheois/LLWP#1] [General]
Hello Luis, 
It seems that the git repo is only your website for LLWP, and the source code is not released. Will the source code be released soon?

**Comment by Ltotheois** [comment]
Hi Luyao,
This is correct, the repository is only for the webpage.
However, you can find the source code under releases or directly via the webpage.
Hope this helps.

**Reply by luyaozou** [reply]
Hello Luis, 
I only find executables (exe or pyw).

**Comment by Ltotheois** [comment]
The .pyw file is the python source code. The difference to .py is only, that the terminal will not be opened. But feel free to change the file extension to .py.

**Reply by luyaozou** [reply]
Oh thank you! I mixed pyc with pyw I thought it was the python binary. My bad
